docs/configuration.md

```diff
@@ -206,6 +206,13 @@ of the most common options to set are:
     name and an array of addresses.
   </td>
 </tr>
+<tr>
+ <td><code>spark.driver.resource.{resourceName}.vendor</code></td>
```

Comment:
Template is harder to use in order to specify the gpu limits as in the example [here](https://kubernetes.io/docs/tasks/manage-gpus/scheduling-gpus/)?

Reply:
I'm not sure what you mean by template is harder to use?  here you specify the spark conf and we translate it into the k8s one

Reply:
The pod template support at the k8s packend allows you to pass any k8s support config option without the need to add spark specific config options.